### PR TITLE
Fix feature gates for v1.28

### PR DESCRIPTION
**Pull request by tengqm**
This PR fixes the feature gate status for v1.28.

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
_sigh_ (it's a shame to need such a large fixup PR)

Thank you very much @tengqm!

/approve

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -784,8 +748,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `UnknownVersionInteroperabilityProxy`: Proxy resource requests to the correct peer kube-apiserver when
   multiple kube-apiservers exist at varied versions.
   See [Mixed version proxy](/docs/concepts/architecture/mixed-version-proxy/) for more information.
-- `UserNamespacesStatelessPodsSupport`: Enable user namespace support for stateless Pods. This flag was renamed on newer releases to `UserNamespacesSupport`.
 - `UserNamespacesSupport`: Enable user namespace support for Pods.
+  This feature was perviously called `UserNamespacesStatelessPodsSupport`
+  before v1.28.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:753]: _perviously_ is a typo
```suggestion
  Before Kubernetes v1.28, this feature gate was named `UserNamespacesStatelessPodsSupport`.
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -732,6 +699,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `SELinuxMountReadWriteOncePod`: Speeds up container startup by allowing kubelet to mount volumes
   for a Pod directly with the correct SELinux label instead of changing each file on the volumes
   recursively. The initial implementation focused on ReadWriteOncePod volumes.
+- `SchedulerQueueingHints`: Enables the scheduler's enhancement called `QueueingHints`,
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:702]:
```suggestion
- `SchedulerQueueingHints`: Enables the scheduler's _queueing hints_ enhancement,
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -591,6 +560,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `KMSv2KDF`: Enables KMS v2 to generate single use data encryption keys.
   See [Using a KMS Provider for data encryption](/docs/tasks/administer-cluster/kms-provider) for more details.
   If the `KMSv2` feature gate is not enabled in your cluster, the value of the `KMSv2KDF` feature gate has no effect.
+- `KubeProxyDrainingTerminatingNodes`: Implement connection draining for
+  terminating nodes for `externalTrafficPolicy: Cluster` services.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:564]:
```suggestion
  terminating nodes for `externalTrafficPolicy: Cluster` Services.
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 4]
```diff
@@ -493,25 +461,26 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
   See [Perform a Rolling Update on a DaemonSet](/docs/tasks/manage-daemon/update-daemon-set/).
-- `DisableAcceleratorUsageMetrics`:
-  [Disable accelerator metrics collected by the kubelet](/docs/concepts/cluster-administration/system-metrics/#disable-accelerator-metrics).
+- `DefaultHostNetworkHostPortsInPodTemplates`: Changes when the default value of
+  `PodSpec.containers[*].ports[*].hostPort`
+  is assigned. The default is to only set a default value in Pods.
+  Enabling this means a default will be assigned even to embedded
+  PodSpecs (e.g. in a Deployment), which is the historical default.
+- `DevicePluginCDIDevices`: Enable support to CDI device IDs in the
+  Device Plugin API.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:470]:
```suggestion
  [Device Plugin](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) API.
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 5]
```diff
@@ -478,6 +445,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   See [Kubelet Checkpoint API](/docs/reference/node/kubelet-checkpoint-api/) for more details.
 - `ContextualLogging`: When you enable this feature gate, Kubernetes components that support
    contextual logging add extra detail to log output.
+- `CronJobsScheduledAnnotation`: Set the scheduled time as an annotation on a CronJob.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:448]:
```suggestion
- `CronJobsScheduledAnnotation`: Set the scheduled job time as an
  {{< glossary_tooltip text="annotation" term_id="annotation" >}} on Jobs that were created
  on behalf of a CronJob.
```

**Comment by reylejano** [comment]
Thank you for the fix
/lgtm

**Pull request merged**